internal/components/components.go

```diff
@@ -299,6 +300,7 @@ func Components() (component.Factories, error) {
 		spanprocessor.NewFactory(),
 		cumulativetodeltaprocessor.NewFactory(),
 		deltatorateprocessor.NewFactory(),
+		schemaprocessor.NewFactory(),
```

Comment:
We typically enable the component after the functionality is working. Let's remove the line for now, can be added after the transform code is written.